src/firmware.cpp

```diff
@@ -12,7 +12,7 @@ static const CommandResponse OK = "";
 // Multiplying by this converts round-trip duration in microseconds to distance to object in millimetres.
 static const float ULTRASOUND_COEFFICIENT = 1e-6 * 343.0 * 0.5 * 1e3;
 
-static const String FIRMWARE_VERSION = "SourceBots PWM/GPIO v2018.4.0";
+static const String FIRMWARE_VERSION = "SourceBots PWM/GPIO v2019.6.0";
```

Comment:
Version number has increased by two.
I assume this board wont have servo support any more so poosibly remove the PWM note.

Reply:
I'd assumed that the convention was `year.month.releasenumber`, but after looking at where the version number came from (f8bdaf6842d602b0cb9948ed347acb2067abd7a2) I now have no clue what the convention is.

Reply:
> I'd assumed that the convention was `year.month.releasenumber`, but after looking at where the version number came from ([f8bdaf6](https://github.com/sourcebots/servo-firmware/commit/f8bdaf6842d602b0cb9948ed347acb2067abd7a2)) I have no clue.

That's what I thought.

@RealOrangeOne can you clarify for us?

Reply:
updated in f8b5dc352b35eda7777af4309be8dcb2157f08ed